resources/__main__.py

```diff
@@ -35,6 +36,7 @@ def main():
     parser.add_argument('actions', nargs='+', choices=(
         'clean',  # clean all resources (assets / data), including book
         'validate',  # validate no resources are changed when re-running
+        'validate_assets',  # manual validation for certain important resources
```

Comment:
Add this as a validation step in CI [here](https://github.com/TerraFirmaCraft/TerraFirmaCraft/blob/a8ef59ef7b89328607047c12132e6bac9afc7e6e/.github/workflows/test.yml#L46)?